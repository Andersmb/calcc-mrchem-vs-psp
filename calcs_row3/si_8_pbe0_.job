#!/bin/bash

#SBATCH --account=nn4654k
#SBATCH --mail-type=None
#SBATCH --job-name=si_8_pbe0_
#SBATCH --output=si_8_pbe0_.log
#SBATCH --error=si_8_pbe0_.err
#SBATCH --time=00-48:00:00
#SBATCH --mem=160GB
#SBATCH --ntasks=30
#SBATCH --cpus-per-task=6

module purge
module load foss/2021b
module load OpenMPI/4.0.3-GCC-9.3.0
module load Python/3.8.2-GCCcore-9.3.0

cp si_8_pbe0_.inp $SCRATCH
cd $SCRATCH
export OMP_NUM_THREADS=6
$HOME/software/mrchem/install_upstream_master/bin/mrchem --launcher 'srun -n 30'  si_8_pbe0_

savefile si_8_pbe0_.out
savefile si_8_pbe0_.json

DIR=/cluster/projects/nn4654k/$(whoami)/MWOrbitals/${SLURM_JOBID}
mkdir -p $DIR
cp orbitals/* $DIR/
echo $DIR > ${SLURM_SUBMIT_DIR}/si_8_pbe0_.orbitals

DIR=/cluster/projects/nn4654k/$(whoami)/MWCheckpoints/${SLURM_JOBID}
mkdir -p $DIR
cp checkpoint/* $DIR/
echo $DIR > ${SLURM_SUBMIT_DIR}/si_8_pbe0_.checkpoint

exit 0