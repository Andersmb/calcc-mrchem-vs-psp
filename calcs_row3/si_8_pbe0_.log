Starting job 5688415 on c2-[34,51],c5-[29,40-42] at Sat May 21 08:29:24 CEST 2022

Copying si_8_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying si_8_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688415      si_8_pbe0_        180                                             00:40:41      0:0 
5688415.bat+      batch         36        1   00:00:03          0   00:00:03   00:40:41      0:0 
5688415.ext+     extern        180        6   00:00:00          1   00:00:00   00:40:41      0:0 
5688415.0      mrchem.x        180       30   00:40:05         29   01:54:29   00:40:12      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688415                                                                          
5688415.bat+     19340K          0     19340K        0              0          0 
5688415.ext+          0          1          0        0              1          0 
5688415.0     15699916K         29 6767806327        0             16          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688415                                                                                               
5688415.bat+       19.44M               0         19.44M        0.07M                0          0.07M 
5688415.ext+        0.00M               1          0.00M            0                1              0 
5688415.0           6.75M               0          6.73M      145.41M                8         77.99M 

Job 5688415 completed at Sat May 21 09:10:04 CEST 2022
