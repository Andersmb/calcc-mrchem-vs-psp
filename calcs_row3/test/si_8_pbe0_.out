                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Si            :        0.739550       -0.000000        0.000000
    1       F            :       -0.364428        2.824762        0.000000
    2       F            :       -0.364428       -1.412381        2.446250
    3       F            :       -0.364428       -1.412381       -2.446250
---------------------------------------------------------------------------
 Center of mass          :       -0.000000       -0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              21
 Beta electrons          :                                              20
 Total electrons         :                                              41
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999909369e-01
    1    1    a          :                              1.000000020832e+00
    2    1    a          :                              1.000000000054e+00
    3    1    a          :                              9.999997934049e-01
    4    1    a          :                              9.999998926636e-01
    5    1    a          :                              9.999999847113e-01
    6    1    a          :                              9.999999899731e-01
    7    1    a          :                              1.000000002366e+00
    8    1    a          :                              1.000000047946e+00
    9    1    a          :                              9.999999038726e-01
   10    1    a          :                              1.000000155501e+00
   11    1    a          :                              9.999990822296e-01
   12    1    a          :                              9.999999245292e-01
   13    1    a          :                              9.999998917503e-01
   14    1    a          :                              1.000000217790e+00
   15    1    a          :                              9.999988024149e-01
   16    1    a          :                              1.000000227279e+00
   17    1    a          :                              9.999997430717e-01
   18    1    a          :                              1.000000041003e+00
   19    1    a          :                              1.000000764497e+00
   20    1    a          :                              1.000000038444e+00
   21    1    b          :                              9.999999860578e-01
   22    1    b          :                              9.999999893082e-01
   23    1    b          :                              9.999999868490e-01
   24    1    b          :                              1.000000001440e+00
   25    1    b          :                              1.000000012008e+00
   26    1    b          :                              9.999999848906e-01
   27    1    b          :                              9.999999901019e-01
   28    1    b          :                              1.000000002366e+00
   29    1    b          :                              1.000000054599e+00
   30    1    b          :                              9.999999171160e-01
   31    1    b          :                              1.000000007898e+00
   32    1    b          :                              9.999996903666e-01
   33    1    b          :                              9.999995222806e-01
   34    1    b          :                              9.999996588194e-01
   35    1    b          :                              1.000000090588e+00
   36    1    b          :                              1.000000078625e+00
   37    1    b          :                              1.000000115398e+00
   38    1    b          :                              9.999997357660e-01
   39    1    b          :                              1.000000016135e+00
   40    1    b          :                              1.000000745763e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                583.744247058532
 E-N energy              :            (au)              -1747.276784578484
 Coulomb energy          :            (au)                455.985547029268
 Exchange energy         :            (au)                -12.721692299267
 X-C energy              :            (au)                -39.497253534057
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                174.304757598474
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -759.765936324008
 Nuclear energy          :            (au)                174.304757598474
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.854611787255e+02
                         :      (kcal/mol)             -3.673824333822e+05
                         :        (kJ/mol)             -1.537128102269e+06
                         :            (eV)             -1.593120956613e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -66.090042778869
    1    1    a          :            (au)                -24.531785236527
    2    1    a          :            (au)                -24.482095890441
    3    1    a          :            (au)                -24.405829557763
    4    1    a          :            (au)                 -5.544568546063
    5    1    a          :            (au)                 -3.880463683105
    6    1    a          :            (au)                 -3.878811141919
    7    1    a          :            (au)                 -3.878125489201
    8    1    a          :            (au)                 -1.207394591688
    9    1    a          :            (au)                 -1.170631982233
   10    1    a          :            (au)                 -1.136318160352
   11    1    a          :            (au)                 -0.563497507012
   12    1    a          :            (au)                 -0.463723876183
   13    1    a          :            (au)                 -0.450950905675
   14    1    a          :            (au)                 -0.444916493042
   15    1    a          :            (au)                 -0.388115265850
   16    1    a          :            (au)                 -0.378006793399
   17    1    a          :            (au)                 -0.372365583152
   18    1    a          :            (au)                 -0.341059916623
   19    1    a          :            (au)                 -0.330103672107
   20    1    a          :            (au)                 -0.256259514704
   21    1    b          :            (au)                -66.085952293289
   22    1    b          :            (au)                -24.528415019739
   23    1    b          :            (au)                -24.478839925104
   24    1    b          :            (au)                -24.402971531468
   25    1    b          :            (au)                 -5.537677561862
   26    1    b          :            (au)                 -3.874710404497
   27    1    b          :            (au)                 -3.874007737539
   28    1    b          :            (au)                 -3.870978874485
   29    1    b          :            (au)                 -1.197844228423
   30    1    b          :            (au)                 -1.161266774343
   31    1    b          :            (au)                 -1.128037472831
   32    1    b          :            (au)                 -0.533446006304
   33    1    b          :            (au)                 -0.448245620285
   34    1    b          :            (au)                 -0.436078375823
   35    1    b          :            (au)                 -0.424876376403
   36    1    b          :            (au)                 -0.379200180211
   37    1    b          :            (au)                 -0.367737797560
   38    1    b          :            (au)                 -0.365915841437
   39    1    b          :            (au)                 -0.331568228255
   40    1    b          :            (au)                 -0.326804373949
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -327.949641209718
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          6.403124e+00        -585.461178725535         -5.854612e+02
    1          4.322987e-01        -588.762087016454         -3.300908e+00
    2          1.794352e-01        -588.899432595305         -1.373456e-01
    3          1.049727e-01        -588.923849235778         -2.441664e-02
    4          5.433940e-02        -588.931939518968         -8.090283e-03
    5          5.594217e-02        -588.937079517883         -5.139999e-03
    6          1.919217e-02        -588.937684968270         -6.054504e-04
    7          1.721925e-02        -588.938067740825         -3.827726e-04
    8          6.696805e-03        -588.938130690449         -6.294962e-05
    9          6.042665e-03        -588.938165515192         -3.482474e-05
   10          2.892201e-03        -588.938176545151         -1.102996e-05
   11          2.264939e-03        -588.938182483249         -5.938098e-06
   12          9.908679e-04        -588.938183893291         -1.410042e-06
   13          4.628722e-04        -588.938184300859         -4.075673e-07
   14          2.495186e-04        -588.938184437019         -1.361607e-07
   15          1.700637e-04        -588.938184487449         -5.042932e-08
   16          7.596408e-05        -588.938184500052         -1.260344e-08
---------------------------------------------------------------------------
                      SCF converged in 16 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Si            :        0.739550       -0.000000        0.000000
    1       F            :       -0.364428        2.824762        0.000000
    2       F            :       -0.364428       -1.412381        2.446250
    3       F            :       -0.364428       -1.412381       -2.446250
---------------------------------------------------------------------------
 Center of mass          :       -0.000000       -0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                587.504984388552
 E-N energy              :            (au)              -1753.943937747585
 Coulomb energy          :            (au)                455.233412376555
 Exchange energy         :            (au)                -12.663378513073
 X-C energy              :            (au)                -39.374022602975
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                174.304757598474
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -763.242942098526
 Nuclear energy          :            (au)                174.304757598474
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.889381845001e+02
                         :      (kcal/mol)             -3.695642874295e+05
                         :        (kJ/mol)             -1.546256979608e+06
                         :            (eV)             -1.602582370908e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -66.369980647638
    1    1    a          :            (au)                -24.797808899376
    2    1    a          :            (au)                -24.797806650947
    3    1    a          :            (au)                -24.797795202308
    4    1    a          :            (au)                 -5.442370263782
    5    1    a          :            (au)                 -3.778691566511
    6    1    a          :            (au)                 -3.775967548666
    7    1    a          :            (au)                 -3.775967429070
    8    1    a          :            (au)                 -1.271811342850
    9    1    a          :            (au)                 -1.247195814028
   10    1    a          :            (au)                 -1.247186243313
   11    1    a          :            (au)                 -0.623379337839
   12    1    a          :            (au)                 -0.550516548979
   13    1    a          :            (au)                 -0.550510742309
   14    1    a          :            (au)                 -0.521090072773
   15    1    a          :            (au)                 -0.497185478691
   16    1    a          :            (au)                 -0.497180851050
   17    1    a          :            (au)                 -0.473399606307
   18    1    a          :            (au)                 -0.473395445269
   19    1    a          :            (au)                 -0.457509325092
   20    1    a          :            (au)                 -0.299026829176
   21    1    b          :            (au)                -66.365957769210
   22    1    b          :            (au)                -24.795120972171
   23    1    b          :            (au)                -24.795118390521
   24    1    b          :            (au)                -24.795108962364
   25    1    b          :            (au)                 -5.436347562953
   26    1    b          :            (au)                 -3.772394285441
   27    1    b          :            (au)                 -3.772393747355
   28    1    b          :            (au)                 -3.768613808266
   29    1    b          :            (au)                 -1.265281039628
   30    1    b          :            (au)                 -1.240636806111
   31    1    b          :            (au)                 -1.240629658395
   32    1    b          :            (au)                 -0.610099256149
   33    1    b          :            (au)                 -0.541395325316
   34    1    b          :            (au)                 -0.541393797498
   35    1    b          :            (au)                 -0.508877521379
   36    1    b          :            (au)                 -0.491431278262
   37    1    b          :            (au)                 -0.491426927760
   38    1    b          :            (au)                 -0.469636089998
   39    1    b          :            (au)                 -0.469631477753
   40    1    b          :            (au)                 -0.455645694154
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -332.072916216656
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.003628       -0.000047        0.000027
 Magnitude               :            (au)                        0.003628
                         :         (Debye)                        0.009221
---------------------------------------------------------------------------
 Nuclear vector          :        0.514153       -0.000000        0.000000
 Magnitude               :            (au)                        0.514153
                         :         (Debye)                        1.306846
---------------------------------------------------------------------------
 Total vector            :        0.517781       -0.000047        0.000027
 Magnitude               :            (au)                        0.517781
                         :         (Debye)                        1.316067
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 40m  7s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
